sdk/samples/iot/aziot_esp8266/aziot_esp8266.ino

```diff
@@ -268,6 +268,19 @@ void setup()
   establishConnection();
 }
 
+static uint8_t m_payload[100];
+static uint32_t m_msgCount = 0;
+static char* getTelemetryPayload()
```

Comment:
Curiosity, since m_msgCount is only used within the getTelemetryPayload(), why not declare it inside the function to keep its scope local?

Reply:
Because we want to keep tracking the count throughout the application lifetime. 
If we define it inside the function, the counter will get lost every time the function is called.

Reply:
not if it is static though.

Reply:
Good job getting the info.
For the sake of keeping the pattern we have used before, I'll keep the variable defined outside the function. I'll move it to the top though.